validator/client/validator.go

```diff
@@ -99,6 +103,15 @@ type validatorStatus struct {
 	index     types.ValidatorIndex
 }
 
+type PrepareBeaconProposalConfig struct {
+	ProposeConfig map[[fieldparams.BLSPubkeyLength]byte]*ValidatorProposerOptions
+	DefaultConfig *ValidatorProposerOptions
+}
+
+type ValidatorProposerOptions struct {
+	FeeRecipient common.Address
+}
```

Comment:
We already have these in the config.go file in the same package, right? If so, could we move those there or use the types that were defined over there?

Reply:
very similar but not the same these uses the byte versions for easier use in our go code. I'll move them together then.